lib/gl_command/context_inspect.rb

```diff
@@ -1,41 +1,58 @@
+# frozen_string_literal: true
+
 module GLCommand
   class ContextInspect
+    PERMITTED_OUTPUTS = %i[string hash].freeze
+
     class << self
       def error(error_obj)
         return '' if error_obj.blank?
 
         error_obj.is_a?(Array) ? error_obj.uniq.join(', ') : error_obj.to_s
       end
 
-      def hash_params(hash)
-        hash.map do |key, value|
-          value_s =
-            if value.nil?
-              'nil'
-            elsif value.respond_to?(:to_sql)
-              object_param_as_sql(value)
-            elsif value.respond_to?(:uuid)
-              object_param_with_id(value, :uuid)
-            elsif value.respond_to?(:id)
-              object_param_with_id(value, :id)
-            else
-              value
-            end
-          "#{key}: #{value_s}"
-        end.join(', ')
+      def hash_params(hash, output: :string)
+        raise unless PERMITTED_OUTPUTS.include?(output)
```

Comment:
Should this raise a specific exception message to specify the allowed output types?

Reply:
Good point!

Added in dc18ccf